README.md

```diff
@@ -12,7 +12,7 @@ Direct dependencies are specified in a top level `versions.props` file and then
 1. Apply the plugin (root project only):
     ```gradle
     plugins {
-        id "com.palantir.consistent-versions" version "1.4.0"
+        id "com.palantir.consistent-versions" version "1.12.4"
```

Comment:
This is going to get stale right away, instead of listing an exact version could we say `<current version>`